src/watchman/models.clj

```diff
@@ -111,6 +112,17 @@
      (k/delete check-statuses (k/where {:host_id host-id :check_id [in check-ids]}))
      (k/delete roles-hosts (k/where {:host_id host-id :role_id role-id})))))
 
+(defn snooze-role [role-id snooze-duration]
```

Comment:
The docstrings here and below are misplaced; they should be before the arg vector.